fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/connection/FHIRDbPropsConnectionStrategy.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.connection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.util.Properties;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.ibm.fhir.persistence.jdbc.exception.FHIRPersistenceDBConnectException;
+import com.ibm.fhir.persistence.jdbc.exception.FHIRPersistenceDataAccessException;
+
+
+/**
+ * Hides the logic behind obtaining a JDBC {@link Connection} from the DAO code.
+ * 
+ * This class implements the old way to obtain DB connections using a {@link Properties} instance
+ */
+@Deprecated
```

Comment:
Can you explain why this new class is already deprecated? 
It seems like it should just not be included or the @Deprecated tag should be removed

Reply:
Yes, this class represents one of the three methods we had for constructing the FHIRPersistenceJDBCImpl. It was used to support getting connections to Derby based on configuration data passed as properties. As the refactor progressed, all the unit tests were updated to use the IConnectionProvider approach. This class was created in case we needed to support the old properties-based connection configuration for any reason. Now the refactor is complete, we can remove it.

Reply:
class has been removed